tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend.robot

```diff
@@ -372,7 +372,20 @@ Update existing data guidance for amendment
 
     user clicks button    Save guidance
 
-# TODO luke: Add footnotes
+Navigate to 'Footnotes' section
+    user clicks link    Footnotes
+    user waits until h2 is visible    Footnotes
+
+Add a Footnote
+    user waits until page contains link    Create footnote
+    user clicks link    Create footnote
+    user waits until h2 is visible    Create footnote
+    user clicks footnote subject radio    Dates test subject    Applies to all data
+    user clicks element    id:footnoteForm-content
+    user enters text into element    id:footnoteForm-content
+    ...    A footnote
+    user clicks button    Save footnote
+    user waits until h2 is visible    Footnotes
```

Comment:
Should we here do a cursory check that the footnote has actually been added at this point?  Could be super simple - just checking that the page contains the new footnote title in a read-only form.